api/nvidia.com/resource/v1beta1/computedomainconfig.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
The 2024 reminded me of the famous quote

> “Every January 1st, an army of open source developers rushes out to update their copyright attributions in licenses and documentation. Why? Because we’ve always done it that way.”

From https://hynek.me/til/copyright-years/.

Boring question, I know... but: do we have a company guideline here?

I researched this topic more than once across projects and companies. I think we can shorten the header and remove the year. See blog post linked above, and also https://github.com/curl/curl/pull/10205

Anyway! :)

Reply:
maybe like  some clothing brands do `Since 1987*` but I am not lawyer, maybe the number on the license header has a deeper legal meaning